src/layer/BlanketOverlay.js

```diff
@@ -0,0 +1,165 @@
+import {Layer} from './Layer.js';
+import * as DomUtil from '../dom/DomUtil.js';
+import * as Util from '../core/Util.js';
+import * as DomEvent from '../dom/DomEvent.js';
+import {Bounds} from '../geometry/Bounds.js';
+
+/*
+ * @class BlanketOverlay
+ * @inherits Layer
+ * @aka L.BlanketOverlay
+ *
+ * Represents an HTML element that covers ("blankets") the entire surface
+ * of the map.
+ *
+ * Do not use this class directly. It's meant for `Renderer`, and for plugins
+ * that rely on one single HTML element
+ */
+
+export const BlanketOverlay = Layer.extend({
+	// @section
+	// @aka BlanketOverlay options
+	options: {
+		// @option padding: Number = 0.1
+		// How much to extend the clip area around the map view (relative to its size)
+		// e.g. 0.1 would be 10% of map view in each direction
+		padding: 0.1,
+
+		// @option continuous: Boolean = false
```

Comment:
This covers the use case of plugins that want to update during pan animations or pinch interactions.

It won't work on `L.Canvas` nor `L.SVG` because `L.Path`s depend a bit too much on the "origin pixel" - I've tried having a look at when those origin-pixel-relative coordinates get updated, but it's a bit too convoluted and there's no quick fix.